apps/framework-cli/src/framework/core/primitive_map.rs

```diff
@@ -207,13 +207,14 @@ impl PrimitiveMap {
         )
         .await?;
 
-        for (config_variable_name, config) in data_models_configs.iter() {
+        // TODO: remove python data models without a config
```

Comment:
probably will refactor it from

1. get all ts/py data models
2. get all ts/py configs
3. combine

to same workflow in ts, but for python

  for each file:
  1. get data models
  2. get configs
  3. remove data models without `moose_data_model`